x-pack/plugins/observability_solution/investigate_app/public/pages/details/components/events_timeline/events_timeline.tsx

```diff
@@ -24,10 +27,19 @@ export const EventsTimeLine = () => {
   const baseTheme = dependencies.start.charts.theme.useChartsBaseTheme();
 
   const { globalParams, updateInvestigationParams } = useInvestigation();
+  const { alert } = useInvestigation();
+
+  const filter = useMemo(() => {
+    const group = (alert?.[ALERT_GROUP] as unknown as Group[])?.find(
+      ({ field }) => field === SERVICE_NAME
+    );
+    return group ? `{"${group.field}":"${alert?.[group.field]}"}` : '';
```

Comment:
```suggestion
    return group ? `{"${SERVICE_NAME}":"${alert?.[SERVICE_NAME]}"}` : '';
```
nit: might be bit more readable